apps/hash-graph/libs/graph/src/knowledge/query.rs

```diff
@@ -134,6 +134,41 @@ pub enum EntityQueryPath<'p> {
     /// [`EntityMetadata`]: graph_types::knowledge::entity::EntityMetadata
     /// [`EntityTemporalMetadata`]: graph_types::knowledge::entity::EntityTemporalMetadata
     CreatedAtDecisionTime,
+    /// The timestamp of the transaction time when the [`Entity`] was _first undrafted_ in the
+    /// database.
```

Comment:
I think some clarification like this would be worth it here and on the decision time field
```suggestion
    /// The timestamp of the transaction time when the first non-draft edition of the [`Entity`] 
    /// was inserted into the database. This may be the first edition, if created as non-draft.
```